### PR TITLE
Create privacy.mdx

### LINKED ISSUE #1262: ✨ New page - Privacy Policy
cc @UlyssesMaclaren @adamcogan @ricksu978 @BrookJeynes 
email thread: **Re: SSW Privacy Policy - Old page**

Hi team, 

We need a new page to replace https://www.ssw.com.au/ssw/company/privacy.aspx

- [ ] Please created a new page with the [content from Cameron](https://sswcom-my.sharepoint.com/:w:/r/personal/cameronshaw_ssw_com_au/Documents/Attachments/Privacy%20Policy%20v%2005.docx?d=w2a725c814f564cd7a2674b4a7e318671&csf=1&web=1&e=emssIn)
- [ ] Make the URL be https://www.ssw.com.au/privacy
- [ ] Make sure the URL in the content is updated accordingly 
- [ ] Add a new section on https://ssw.com.au/terms-and-conditions with a link to the new page (E.g. "32 - Privacy Policy")
- [ ] Get a 'test pass' from Tiago
- [ ] Redirect https://www.ssw.com.au/ssw/company/privacy.aspx. to the new page

### PR DESCRIPTION
**Pull request by tiagov8**
cc @adamcogan @UlyssesMaclaren @BenDunkerleySSW @bradystroud @BrookJeynes @ricksu978 

- Affected routes: /privacy

- Fixed https://github.com/SSWConsulting/SSW.Website/issues/1262

As per [Privacy Policy v 05.docx](https://aus01.safelinks.protection.outlook.com/ap/w-59584e83/?url=https%3A%2F%2Fsswcom-my.sharepoint.com%2F%3Aw%3A%2Fg%2Fpersonal%2Fcameronshaw_ssw_com_au%2FEYFccipWT9dMomdLSn4xhnEBPU2jGMu2AOVyYQKuarqWNw&data=05%7C01%7CTiagoAraujo%40ssw.com.au%7C028dc47177454c90a2a608dbb3e94f66%7Cac2f7c34b93548e9abdc11e5d4fcb2b0%7C0%7C0%7C638301586869048740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=YYy2of1nliZVj3%2B8s%2BOFJXcXXvwrORP%2FRZ9YvzzA7h8%3D&reserved=0)

### CODE REVIEW

File: content/pages/privacy.mdx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+You can configure your internet browser to accept all cookies, reject all cookies or notify you when a cookie is sent. Please refer to your internet browser’s instructions to learn more about these functions. Most Web browsers automatically accept cookies, but you can usually modify your browser settings to decline cookies if you prefer. If you choose to decline cookies, you may not be able to fully experience the interactive features of this site.
+
+#### Why SSW use cookies
+
+This site uses cookies in order to:
+
+- remember your preferences for using this site;
+-	manage the signup process when you create an account with us;
+-	recognise you as logged in while you remain so. This avoids your having to log in again every time you visit a new page;
+-	facilitate e-commerce transactions, to ensure that your order is remembered betSSWen pages during the checkout process;
```
Comment (bradystroud) [inline content/pages/privacy.mdx]: This looks like a typo but I don't know what the correct text would be.

**Review by bradystroud — COMMENTED** [review]
I notice we are using Australian spelling everywhere on this page, is this some legal requirement or can we make it American as per https://ssw.com.au/rules/spelling-do-you-use-us-english/

**Review by bradystroud — CHANGES_REQUESTED** [review]
see comments above

File: content/pages/privacy.mdx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+To ensure that your personal information is accurate and up to date, please promptly advise us of any changes to your information by contacting our data protection officer by email or by post.
+
+### Changes to this Privacy Policy
+
+From time to time, it may be necessary for us to revise this Privacy Policy. Any changes will be in accordance with any applicable requirements under the Privacy Act and the Australian Privacy Principles. SSW may notify you about changes to this Privacy Policy by posting an updated version on this site.
+
+***
+
+If you require any further information about the Privacy Act and the Australian Privacy Principles, you can visit the Federal Privacy Commissioner’s Website (see [www.privacy.gov.au](www.privacy.gov.au)).
```
Comment (ricksu978) [inline content/pages/privacy.mdx]: Have we linked to the correct page? SSL has expired on `https://www.privacy.gov.au`

image
Figure: SSL Expiry warning from Chrome

Comment (bradystroud) [reply content/pages/privacy.mdx]: Maybe its supposed to be this page 

https://www.oaic.gov.au/privacy

**Comment by bradystroud** [comment]
> I notice we are using Australian spelling everywhere on this page, is this some legal requirement or can we make it American as per https://ssw.com.au/rules/spelling-do-you-use-us-english/

As per my teams chat with @BenDunkerleySSW, Australian English could be used because it’s in accordance with Australian law and the wording of the act

File: content/pages/privacy.mdx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+You can configure your internet browser to accept all cookies, reject all cookies or notify you when a cookie is sent. Please refer to your internet browser’s instructions to learn more about these functions. Most Web browsers automatically accept cookies, but you can usually modify your browser settings to decline cookies if you prefer. If you choose to decline cookies, you may not be able to fully experience the interactive features of this site.
+
+#### Why SSW use cookies
+
+This site uses cookies in order to:
+
+* remember your preferences for using this site;
+* manage the signup process when you create an account with us;
+* recognise you as logged in while you remain so. This avoids your having to log in again every time you visit a new page;
+* facilitate e-commerce transactions, to ensure that your order is remembered betSSWen pages during the checkout process;
```
Comment (BenDunkerleySSW) [inline content/pages/privacy.mdx]: As per my conversation with Cameron, the text on line 74 is a typo. The text should be changed: 
from: 
        'is remembered betSSWen pages' 
to: 
        'is remembered between pages'

**Review by BenDunkerleySSW — CHANGES_REQUESTED** [review]

**Comment by BenDunkerleySSW** [comment]
As per my conversation with Cameron, the text can be changed to read American English as it will not affect the meaning of any terms. This was it is compliant with the SSW rules about US English.

File: content/pages/privacy.mdx [hunk 4]
```diff
@@ -0,0 +1,206 @@
+---
```
Comment (BenDunkerleySSW) [inline content/pages/privacy.mdx:1]: As per my conversation with Cameron, the policy can be changed to read in American English, as the meaning of the terms within will not be affected by spelling. This way the document can be compliant with SSW rules regarding US english.

**Review by BenDunkerleySSW — CHANGES_REQUESTED** [review]

**Review by bradystroud — APPROVED** [review]

File: content/pages/privacy.mdx (hunk 3 above)
Comment (bradystroud) [reply content/pages/privacy.mdx]: Fixed

File: content/pages/privacy.mdx (hunk 2 above)
Comment (bradystroud) [reply content/pages/privacy.mdx]: Fixed

**Review by BenDunkerleySSW — APPROVED** [review]

**Pull request merged**